pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -1078,6 +1076,12 @@ def process_circuits(
                             ),
                         )
                     ).decode("utf-8")
+
+                    for name, count in Counter(
```

Comment:
qasm only excludes unused scratch bits, I have now decided to exclude all of them here for QIR. I am not sure if that is the right thing to do, but I would assume that no users are interested in seeing this?
Open to discussions.

Reply:
Maybe QASM should exclude all of them as well?